src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-topic-details/rgw-topic-details.component.html

```diff
@@ -0,0 +1,81 @@
+  <ng-container *ngIf="!!selection">
+    <cds-tabs type="contained"
+              theme="light">
+      <cds-tab heading="Destination Endpoints">
```

Comment:
What about just `Endpoints` as the heading
```suggestion
      <cds-tab heading="Endpoints">
```

Reply:
"Heading suggested by Q.E."